rpcs3/rpcs3qt/debugger_frame.cpp

```diff
@@ -473,9 +493,58 @@ void debugger_frame::EnableButtons(bool enable)
 
 void debugger_frame::ClearBreakpoints()
 {
+	//deleting the breakpoints here turns this function into an all purpose clear breakpoints instead of just clearing on emu stop
```

Comment:
minor quibble, you have 2 empty lines underneath this comment before the for loop.  most of the code base would have no lines, with comment directly above denoted loop.

Reply:
Ah. I'll check more carefully next time :)

Reply:
Remove the blank lines

Reply:
Oh I didn't realize that I was the only one that could make the changes. I'm new to github. 1 sec

Reply:
Technically I can edit the changeset for you but its better if the author does it themselves. That way, commit history is intact and git blame identifies the authors correctly.